00_Core/Scripts/Source/MilkMCM.psc

```diff
@@ -3231,7 +3231,7 @@ endState
 state Debug_MM_MilkMax_Basevalue_Slider
 	event OnSliderOpenST()
 		SetSliderDialogStartValue(MME_Storage.getMilkMaxBasevalue(MaidlistA[MaidIndex]))
-		SetSliderDialogDefaultValue(0)
+		SetSliderDialogDefaultValue(2)
 		SetSliderDialogRange(0, 20)
```

Comment:
I intentionally set it to 0 (and also allowed for a negative scaling factor to enable the players to set up anything they want. Maybe they want to completely disable the fixed base value and go for a higher level-dependent scaling or maybe to start with a high base value and define no (or even negative) scaling?